File: javascript_games/gullible.js
/*
Gullible, by Al Sweigart al@inventwithpython.com
How to keep a gullible person busy for hours. (This is a joke program.)
*/

'use strict';

const readlineSync = require('readline-sync');

console.log('Gullible, by Al Sweigart al@inventwithpython.com');

while (true) {  // Main program loop.
    console.log('Do you want to know how to keep a gullible person busy for hours?');
    let response = readlineSync.question('> ');  // Get the user's response.
    if (response.toLowerCase() === 'no' || response.toLowerCase() === 'n') {
        break;  // If "no", break out of this loop.
    }
    if (response.toLowerCase() === 'yes' || response.toLowerCase() === 'y') {
        continue;  // If "yes", continue to the start of this loop.
    }
    process.stdout.write('"');
    process.stdout.write(response);
    process.stdout.write('" is not a valid yes/no response.\n');
}

console.log('Thank you. Have a nice day!');

File: src/gamesbyexample/gullible.py
"""Gullible, by Al Sweigart al@inventwithpython.com
How to keep a gullible person busy for hours. (This is a joke program.)
This code is available at https://nostarch.com/big-book-small-python-programming
Tags: tiny, beginner, humor"""
__version__ = 0
print('Gullible, by Al Sweigart al@inventwithpython.com')

while True:  # Main program loop.
    print('Do you want to know how to keep a gullible person busy for hours? Y/N')
    response = input('> ')  # Get the user's response.
    if response.lower() == 'no' or response.lower() == 'n':
        break  # If "no", break out of this loop.
    if response.lower() == 'yes' or response.lower() == 'y':
        continue  # If "yes", continue to the start of this loop.
    print('"{}" is not a valid yes/no response.'.format(response))

print('Thank you. Have a nice day!')
